pkg/adapter/v2/test/test_client.go

```diff
@@ -71,10 +84,13 @@ func (c *TestCloudEventsClient) Sent() []cloudevents.Event {
 	return r
 }
 
-func NewTestClient() *TestCloudEventsClient {
+func NewTestClient(reporter source.StatsReporter) *TestCloudEventsClient {
```

Comment:
there is no reason to test that the test cloudevents client uses the test stats reporter.